ui/app/src/models/Version.ts

```diff
@@ -14,7 +14,7 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { LegacyItem } from './Item';
+import { LegacyItem, SandboxItem } from './Item';
```

Comment:
Check terminal warnings. Are you still using LegacyItem here for example?

Reply:
yes